launch/Jenkinsfile

```diff
@@ -16,25 +16,25 @@ try {
     
     node {
         deleteDir()
-        releaseOpenHab2Component('openhab-core', false)
+        releaseComponent('openhab-core', false, "master")
     }
     
     // checkpoint "openHAB Core released"
     
     
     node {
         deleteDir()
-        releaseOpenHab2Component('openhab-addons', true)
-        releaseOpenHab2Component('org.openhab.binding.zwave', true)
-        releaseOpenHab2Component('org.openhab.binding.zigbee', true)
-        releaseOpenHab2Component('openhab-webui', true)
+        releaseComponent('openhab-addons', true, env.OH_BRANCH)
+        releaseComponent('org.openhab.binding.zwave', true, "master")
+        releaseComponent('org.openhab.binding.zigbee', true, "master")
```

Comment:
This will fail unless @cdjackson updates the code to be compatible with OH3.

Reply:
That's correct, but he planned to make the code compile within the next days, so I kept it in here.

Reply:
OK, let's then merge it and see if all goes well until it reaches the ZigBee component. :wink: